digdag-standards/src/main/java/io/digdag/standards/command/kubernetes/DefaultKubernetesClient.java

```diff
@@ -133,22 +142,57 @@ protected Container createContainer(final CommandContext context, final CommandR
                 .build();
     }
 
+    protected JsonNode convertConfigToJsonNode(final Config config, String key) {
```

Comment:
@muga 
I have a question about this method.
This method purpose is same to bellow method.
https://github.com/treasure-data/digdag/blob/a7039bc76c3de08022411b8897517757bec42172/digdag-client/src/main/java/io/digdag/client/config/Config.java#L385
So I' like to make `Config.tryParse Nested` method to public if you allow that.